File: app/src/main/java/com/kstechsolutions/mvvm/modules/comments/model/Comment.java
package com.kstechsolutions.mvvm.modules.comments.model;

import java.io.Serializable;

/**
 * Created by muhammadumairshafique on 16/04/2018.
 */

public class Comment implements Serializable {
    private String name;
    private String email;
    private String body;

    public String getName() {
        return name;
    }

    public String getEmail() {
        return email;
    }

    public String getBody() {
        return body;
    }
}

File: app/src/main/java/com/kstechsolutions/mvvm/modules/comments/Comment.kt
package com.kstechsolutions.mvvm.modules.comments

/**
 * Created by muhammad umair shafique on 29/09/2019.
 */

data class CommentUIModel(val isLoading: Boolean = false, val data: List<Comment> = emptyList(), val errorMessage: String = "")

data class Comment(val name: String = "", val email: String = "", val body: String = "")
